app/furniture/marketplace/product_component.html.erb

```diff
@@ -1,18 +1,22 @@
 <%= render CardComponent.new(dom_id: dom_id(product)) do |card| %>
-  <%- card.with_header(variant: :no_padding) do %>
-    <% if product.photo.present? %>
+  <% if product.photo.present? %>
+    <%- card.with_header(variant: :no_padding) do %>
       <figure>
         <%= image_tag hero_image, class: "rounded-t-lg w-full" %>
         <figcaption class="px-2 pt-4 sm:px-4">
           <h3><%= name %></h3>
+          <p class="text-sm">Serves <%= product.servings %></p>
           <%- if product.archived? %>
             <span class="italic">(archived)</span>
           <%- end %>
         </figcaption>
       </figure>
-    <%- else %>
-      <h3 class="px-4"><%= name %></h3>
     <% end %>
+  <%- else %>
+    <%- card.with_header do %>
+      <h3><%= name %></h3>
+      <p class="text-sm">Serves <%= product.servings %></p>
+    <%- end %>
   <%- end %>
```

Comment:
Or maybe *all* these lines want to be the `Marketplace::Product::HeaderComponent`?